lib/dry/types/builder_methods.rb

```diff
@@ -83,7 +83,7 @@ def Constructor(klass, cons = nil, &block)
         Definition.new(klass).constructor(cons || block || klass.method(:new))
       end
 
```

Comment:
We probably going to need a builder method for building `Map`

```ruby
def Map(options={})
  Map.new(options)
end
```

Reply:
seems reasonable, I will go ahead and add it

Reply:
done